rules/subrepo_rules.build_defs

```diff
@@ -206,13 +210,17 @@ def github_repo(name:str, repo:str, revision:str, build_file:str=None, hashes:st
 
     return new_http_archive(
         name = name,
-        urls = [f'https://github.com/{org}/{repo}/archive/{revision}.zip'],
+        urls = [
+            #f'https://github.com/{org}/{repo}/archive/{revision}.zip',
```

Comment:
With this line commented out the problem is reproducible. When running ./bootstrap.sh it fails downloading third_party/cc/gtest. The zip file is downloaded, but it does not get extracted. To complete this PR I would want to uncomment this line to maintain backward compatibility and make the change a lower risk.